src/components/textInput/textInput.tsx

```diff
@@ -1,9 +1,14 @@
 import './textInput.css'
 
-import SendIcon from '@mui/icons-material/Send'
+import MicNoneRoundedIcon from '@mui/icons-material/MicNoneRounded'
+import MicRoundedIcon from '@mui/icons-material/MicRounded'
```

Comment:
Please use material symbols instead

Reply:
Since filled icons are used, these are the updated icons:
Not recording:
<img width="298" alt="Screenshot 2024-04-19 at 10 10 24 AM" src="https://github.com/rustic-ai/ui-components/assets/111031789/466fd04c-3341-4cda-bee4-077cdc528697">

Recording: 
<img width="298" alt="Screenshot 2024-04-19 at 10 10 29 AM" src="https://github.com/rustic-ai/ui-components/assets/111031789/35bf9d0b-bcb3-4865-be25-a6c2243f0905">